docker-compose.yml

```diff
@@ -112,41 +112,42 @@ services:
-      HASURA_GRAPHQL_URL: http://hasura-proxy:${HASURA_PROXY_PORT}/v1/graphql
-      HASURA_METADATA_URL: http://hasura-proxy:${HASURA_PROXY_PORT}/v1/metadata
-      HASURA_SCHEMA_URL: http://hasura-proxy:${HASURA_PROXY_PORT}/v2/query
-      JWT_SECRET: ${JWT_SECRET}
-      GOOGLE_CLIENT_ID: ${GOOGLE_CLIENT_ID}
-      GOOGLE_CLIENT_SECRET: ${GOOGLE_CLIENT_SECRET}
+      AIRBRAKE_PROJECT_ID: ${AIRBRAKE_PROJECT_ID}
+      AIRBRAKE_PROJECT_KEY: ${AIRBRAKE_PROJECT_KEY}
       API_URL_EXT: ${API_URL_EXT}
-      EDITOR_URL_EXT: ${EDITOR_URL_EXT}
-      SESSION_SECRET: ${SESSION_SECRET}
-      AWS_S3_REGION: ${AWS_S3_REGION}
+      APP_ENVIRONMENT: ${APP_ENVIRONMENT}
       AWS_ACCESS_KEY: ${AWS_ACCESS_KEY}
-      AWS_SECRET_KEY: ${AWS_SECRET_KEY}
-      AWS_S3_BUCKET: ${AWS_S3_BUCKET}
       AWS_S3_ACL: ${AWS_S3_ACL}
-      PORT: ${API_PORT}
+      AWS_S3_BUCKET: ${AWS_S3_BUCKET}
+      AWS_S3_REGION: ${AWS_S3_REGION}
+      AWS_SECRET_KEY: ${AWS_SECRET_KEY}
       BOPS_API_TOKEN: ${BOPS_API_TOKEN}
-      AIRBRAKE_PROJECT_ID: ${AIRBRAKE_PROJECT_ID}
-      AIRBRAKE_PROJECT_KEY: ${AIRBRAKE_PROJECT_KEY}
-      UNIFORM_TOKEN_URL: ${UNIFORM_TOKEN_URL}
-      UNIFORM_SUBMISSION_URL: ${UNIFORM_SUBMISSION_URL}
-      GOVUK_NOTIFY_API_KEY: ${GOVUK_NOTIFY_API_KEY}
-      HASURA_PLANX_API_KEY: ${HASURA_PLANX_API_KEY}
-      FILE_API_KEY: ${FILE_API_KEY}
-      FILE_API_KEY_NEXUS: ${FILE_API_KEY_NEXUS}
-      FILE_API_KEY_BARNET: ${FILE_API_KEY_BARNET}
-      SLACK_WEBHOOK_URL: ${SLACK_WEBHOOK_URL}
-      ORDNANCE_SURVEY_API_KEY: ${ORDNANCE_SURVEY_API_KEY}
-      MINIO_PORT: ${MINIO_PORT}
       CORS_ALLOWLIST: ${EDITOR_URL_EXT}, ${API_URL_EXT}
+      EDITOR_URL_EXT: ${EDITOR_URL_EXT}
       ENCRYPTION_KEY: ${ENCRYPTION_KEY}
+      FILE_API_KEY_BARNET: ${FILE_API_KEY_BARNET}
+      FILE_API_KEY_NEXUS: ${FILE_API_KEY_NEXUS}
+      FILE_API_KEY: ${FILE_API_KEY}
+      GOOGLE_CLIENT_ID: ${GOOGLE_CLIENT_ID}
+      GOOGLE_CLIENT_SECRET: ${GOOGLE_CLIENT_SECRET}
+      GOVUK_NOTIFY_API_KEY: ${GOVUK_NOTIFY_API_KEY}
+      HASURA_GRAPHQL_ADMIN_SECRET: ${HASURA_GRAPHQL_ADMIN_SECRET}
+      HASURA_GRAPHQL_URL: http://hasura-proxy:${HASURA_PROXY_PORT}/v1/graphql
+      HASURA_METADATA_URL: http://hasura-proxy:${HASURA_PROXY_PORT}/v1/metadata
+      HASURA_PLANX_API_KEY: ${HASURA_PLANX_API_KEY}
+      HASURA_SCHEMA_URL: http://hasura-proxy:${HASURA_PROXY_PORT}/v2/query
       IDOX_NEXUS_CLIENT: ${IDOX_NEXUS_CLIENT}
-      IDOX_NEXUS_TOKEN_URL: ${IDOX_NEXUS_TOKEN_URL}
       IDOX_NEXUS_SUBMISSION_URL: ${IDOX_NEXUS_SUBMISSION_URL}
+      IDOX_NEXUS_TOKEN_URL: ${IDOX_NEXUS_TOKEN_URL}
+      JWT_SECRET: ${JWT_SECRET}
+      MAPBOX_ACCESS_TOKEN: ${MAPBOX_ACCESS_TOKEN}
+      MINIO_PORT: ${MINIO_PORT}
+      ORDNANCE_SURVEY_API_KEY: ${ORDNANCE_SURVEY_API_KEY}
+      PORT: ${API_PORT}
+      SESSION_SECRET: ${SESSION_SECRET}
+      SLACK_WEBHOOK_URL: ${SLACK_WEBHOOK_URL}
+      UNIFORM_SUBMISSION_URL: ${UNIFORM_SUBMISSION_URL}
+      UNIFORM_TOKEN_URL: ${UNIFORM_TOKEN_URL}
```

Comment:
ran "sort lines ascending" on these secrets for readability - only addition is `MAPBOX_ACCESS_TOKEN` :+1: